prediction_market_agent/agents/blockchain_coding_agent/app.py

```diff
@@ -1,51 +1,57 @@
 import asyncio
+from enum import Enum
 
 import streamlit as st
-from autogen import Cache
 
-from prediction_market_agent.agents.blockchain_coding_agent.agents import (
-    get_code_executor_agent,
-    get_code_writer_agent,
+from prediction_market_agent.agents.blockchain_coding_agent.agents import get_agent_team
+from prediction_market_agent.agents.blockchain_coding_agent.prompts import (
+    message_read,
+    message_write,
```

Comment:
Keep the practice of  global variables being upper case please

Reply:
Renamed